packages/devtools/lib/src/ui/flutter_html_shim.dart

```diff
@@ -14,7 +14,14 @@ String fontStyleToCss(TextStyle textStyle) {
   return sb.toString();
 }
 
+final Map<Color, String> cssColors = {};
+
 String colorToCss(Color color) {
+  if (cssColors.containsKey(color)) {
```

Comment:
optimize for cache hits so instead write:
String cssColor = ccsColors[color];
if (cssColor != null) return cssColor;

Reply:
done.